@harmut01 CI errored early

```

[2021-02-22T12:34:58.040Z] multiprocessing.pool.RemoteTraceback: 

[2021-02-22T12:34:58.040Z] """

[2021-02-22T12:34:58.040Z] Traceback (most recent call last):

[2021-02-22T12:34:58.040Z]   File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker

[2021-02-22T12:34:58.040Z]     result = (True, func(*args, **kwds))

[2021-02-22T12:34:58.040Z]   File "/usr/lib/python3.6/multiprocessing/pool.py", line 44, in mapstar

[2021-02-22T12:34:58.040Z]     return list(map(*args))

[2021-02-22T12:34:58.040Z]   File "smart-tester/target_toolchain_selector.py", line 92, in get_resources_filenames_target_toolchains

[2021-02-22T12:34:58.040Z]     resources_filenames_target_toolchains[target][toolchain] = get_resources_filenames(".", target, toolchain)

[2021-02-22T12:34:58.040Z]   File "smart-tester/target_toolchain_selector.py", line 32, in get_resources_filenames

[2021-02-22T12:34:58.040Z]     notify=MockNotifier())

[2021-02-22T12:34:58.040Z]   File "./tools/build_api.py", line 514, in prepare_toolchain

[2021-02-22T12:34:58.040Z]     target, notify, macros, build_dir=build_dir, build_profile=profile, coverage_patterns=coverage_patterns)

[2021-02-22T12:34:58.040Z]   File "./tools/toolchains/iar.py", line 64, in __init__

[2021-02-22T12:34:58.040Z]     self.check_c_lib_supported(target, "iar")

[2021-02-22T12:34:58.040Z]   File "./tools/toolchains/mbed_toolchain.py", line 1113, in check_c_lib_supported

[2021-02-22T12:34:58.040Z]     UNSUPPORTED_C_LIB_EXCEPTION_STRING.format(target.c_lib)

[2021-02-22T12:34:58.040Z] tools.utils.NotSupportedException: std C library option not supported for this target.
```

Please review. I cant find in the logs what "this target" is it

--- a/targets/targets.json
+++ b/targets/targets.json
@@ -4808,8 +4808,7 @@
         "core": "Cortex-A9",
         "supported_toolchains": [
             "ARM",
-            "GCC_ARM",
-            "IAR"
+            "GCC_ARM"
         ],
         "extra_labels": [
             "RENESAS",
@@ -4941,7 +4940,7 @@
         "inherits": ["Target"],
         "public": false,
         "core": "Cortex-A9",
-        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "supported_toolchains": ["ARM", "GCC_ARM"],
         "extra_labels": ["RENESAS", "RZ_A2XX"],
         "device_has": [
             "SLEEP",

